spec/tools/miq_config_sssd_ldap/miqldap_configuration_spec.rb

```diff
@@ -17,8 +17,8 @@
     end
 
     it 'does not merge current authentication setting with options when doing a fresh configuration' do
-      expect(described_class).to_not receive(:current_authentication_settings)
-      described_class.new(options)
+      config = described_class.new(options)
+      expect(config).to_not receive(:current_authentication_settings)
```

Comment:
the `.new` isn't going to work here since any other instance won't get called.

I think `any_instance_of(described_class)` is the proper usage. of course, rubocop will freak... but it is the proper usage.

Reply:
I guess I don’t understand well enough what the test is trying to test. I’ll switch it when i get back.